docs/usage.rst

```diff
@@ -4,125 +4,87 @@ Usage
 Create a Kubernetes cluster
 ---------------------------
 
-Run OpenStack magnum docker client:
+In order to create a Kubernetes cluster using OpenStack infrastructure,
+we need to run the magnum command, which we can have available running
+the magnum docker client:
 
 ::
 
-$ docker login gitlab-registry.cern.ch
-$ sudo docker run -it gitlab-registry.cern.ch/cloud/ciadm /bin/bash
+  $ docker login gitlab-registry.cern.ch
+  $ sudo docker run -it gitlab-registry.cern.ch/cloud/ciadm /bin/bash
 
-Once this is done, follow the `Cloud
-Docs <http://clouddocs.web.cern.ch/clouddocs/containers/quickstart.html#create-a-cluster>`__
-to create a Kubernetes cluster.
+Or logging into `lxplus-cloud`:
 
-Download the code
------------------
-Get git repo.
-
-``git clone --recursive https://github.com/focilo/focilo.git``
-
-And change Kubernetes service account token in step-broker node.
 ::
 
-    $ kubectl get secrets
-    NAME                  TYPE                                  DATA      AGE
-    default-token-XXXXX   kubernetes.io/service-account-token   3         13d
-    $ sed 's/default-token-02p0z/default-token-XXXXX/' -i kubernetes-cluster/deployments/cap-system/step-broker-rc.yaml
-
-Create the Kubernetes resources using manifest files
-----------------------------------------------------
-
--  Cap system instances:
-
-   ``kubectl create -f kubernetes-cluster/deployments/cap-system``
+  $ ssh lxplus-cloud.cern.ch
 
--  Yadage workers (currently only one):
+Once we have it available we are ready to create the cluster:
 
-   ``kubectl create -f kubernetes-cluster/deployments/yadage-workers/``
+::
 
--  Fibonacci worker (Since Yadage workers are not launching jobs yet, add a fibonacci worker to see how it works).
+  $ magnum cluster-create --name reana-cluster --keypair-id reanakey \
+                          --cluster-template kubernetes --node-count 2
 
-   ``kubectl create -f kubernetes-cluster/deployments/fibonacci-workers/alice-worker-rc.yaml``
+Lastly, we must load the cluster configuration into the Kubernetes
+client:
 
--  Services:
+::
```

Comment:
Instead of empty `::`, you can use something like

```
.. code-block:: console
```
or 

```
.. code-block:: shell
```

(just some cosmetics)